target/linux/ath79/image/generic.mk

```diff
@@ -583,6 +583,16 @@ define Device/comfast_cf-e314n-v2
 endef
 TARGET_DEVICES += comfast_cf-e314n-v2
 
+define Device/comfast_cf-e320n-v2
+  SOC := qca9531
+  DEVICE_VENDOR := COMFAST
+  DEVICE_MODEL := CF-E320N
+  DEVICE_VARIANT := v2
+  DEVICE_PACKAGES := rssileds kmod-leds-gpio kmod-usb-core kmod-usb2
```

Comment:
kmod-usb-core is selected automatically and can be dropped here.

And I don't think we need kmod-leds-gpio either.

Reply:
Hmm, and I don't see USB here at all ...

Reply:
I removed the USB and GPIO modules